app/pages/_app.tsx

```diff
@@ -59,6 +59,10 @@ export default class App extends NextApp<AppProps> {
       ? JSON.parse(String(router.query.relayVars))
       : {};
 
+    const pageVars = router.query.pageVars
+      ? JSON.parse(String(router.query.pageVars))
+      : {};
+
```

Comment:
we don't need an extra `pageVars`, the pagination variables could be in `relayVars`. This would allow us to make the fact that `offset` and `max_results` are reserved variable names more explicit (pageVars overrides relayVars below)

Reply:
Nevermind, relayVars are reset to an empty object when clearing search filters